File: Python Word Examples/TextRangesApiDemo/updateTextRangeFormat.py
import spirecloudword
from spirecloudword import TextRangeFormat, Color, Font
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.TextRangesApi(configuration)

file_name = 'Template.docx'
paragraph_path = "Section/0/Body/0/Paragraph/1"
index = 0
text_range_format = TextRangeFormat()
text_range_format._text = 'E-iceblue'
text_range_format._font = Font(40, 'Cooper Black', Color(100, 100, 100))
extra = {'folder': "input", 'dest_file_path': 'output/UpdateTextRangeFormat.docx'}
result = api.update_text_range_format(file_name, paragraph_path, index, text_range_format, **extra)

File: JavaScript Word Examples/TextRangeApiDemo/updateTextRangeFormat.js
(function (){
    var appId = "your id";
    var appKey = "your key"; 
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword= require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.TextRangesApi(apiClient);

    var name = "Template.docx";
    var paragraphPath = "Section/0/Body/0/Paragraph/0";
    var index = 0;
    var textRange = new Spirecloudword.TextRangeFormat();
    textRange.font = new Spirecloudword.Font();
    textRange.font.fontSize = 30.0;
    textRange.font.fontName = "Aleo";
    textRange.font.color = new Spirecloudword.Color();
    textRange.font.color.red = 0;
    textRange.font.color.green = 255;
    textRange.font.color.blue = 255;
    var destFilePath = "output/UpdateTextRangeFormat.docx";
    var opt = {
        folder: 'input',
        indexInParagraph: null,
        storage: '',
        password: ''
    }
    
    instance.updateTextRangeFormat(name, paragraphPath, index, textRange, destFilePath, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();